### ISSUE TITLE
[Prio 0] Account for different python string types during setup_code_coverage

### ISSUE THREAD

**Issue by JacobCallahan** [SatelliteQE/automation-tools#654]
During our latest RHEL 7 ISO installation attempt, the installer ran into this issue.

```
run: mkdir -p /etc/coverage
Traceback (most recent call last):
...
  File "/home/jenkins/workspace/provisioning-6.2-rhel7/automation_tools/__init__.py", line 737, in setup_code_coverage
    coveragerc_file.write('[run]\n')
TypeError: unicode argument expected, got 'str'
```

I did some quick searching and found a possible solution

```python
try:
    # Python 2
    from cStringIO import StringIO
except ImportError:
    # Python 3
    from io import StringIO
```
Then remove ```io.``` from the rest of the code base.

**Comment by JacobCallahan** [comment]
Looking at our code, it looks like that solution above probably won't work, since we try to import from essentially the same module.

**Comment by kbidarkar** [comment]
Fixed in #655
